core/spec.md

```diff
@@ -2677,11 +2685,16 @@ The following describes the attributes of Registry model:
     it is valid for an implementation to only support one (`1`) Version when
     `maxversions` is set to `0`.
   - When the limit is exceeded, implementations MUST prune Versions by
-    deleting the oldest Version (based on creation times) first, skipping the
-    Version marked as "default". An exception to this pruning rule is if
-    `maxversions` value is one (`1`) then the newest Version of the Resource
-    MUST always be the "default" and the `setdefaultversionsticky` aspect
-    MUST be `false`.
+    deleting the oldest Version first, skipping the
+    Version marked as "default". The strategy for finding the oldest Version
```

Comment:
I think it might be good to add a section to the primer to talk about how "oldest" doesn't necessarily mean "the one with the oldest createdat timestamp". Normally, that's what we'd expect the main algorithm to focus on, but  "ancestors" take higher priority because they more accurately describe an ordered lineage than timestamps do.

Reply:
I added a section, it's possibly a bit too detailed in the sense that it repeats some of the info below, but it felt a bit odd to link to the maxversion part here.